### ISSUE TITLE
List of errorcodes in wlfreerdp

### ISSUE THREAD

**Issue by pbuchholz123** [FreeRDP/FreeRDP#9516]
Hello,

we moved from xfreerdp to wlfreerdp due to Debian 12 Bookworm.

Version:
wlfreerdp --version

Output: This is FreeRDP version 2.11.2 (2.11.2)

I have a script running and doing stuff based on the error code xfreerdp returned.
This is the list i used for that, which to be fair, has some mistakes...

https://github.com/FreeRDP/FreeRDP/blob/2.11.2/client/X11/xfreerdp.h (Line 282-348 - enum XF_EXIT_CODE)
=> for example, xfreerdp actually returnd exit code 12 for EXIT_LOGOFF, not 2

But with wlfreerdp it seems that this list is unusable, because for the most of the time i receive error code 255 for whatever reason...
=> for example, xfreerdp returned 134 if my password is wrong, now it is 255

Where can i find an updated list of the exit codes?

Thank you in advance

Cheers

**Comment by akallabeth** [comment]
better stay with `xfreerdp`, `wlfreerdp` does lack a lot of features (like this one).
we´re deprecating it with `FreeRDP 3.0` in favour of `sdl-client`, which already can do more

**Comment by pbuchholz123** [comment]
Hello @akallabeth 

thanks for your reply.

We used wlfreerdp because it has way better performance than xfreerdp.

I am trying to use the new sdl-client, but i have a few questions..

I followed theese instructions: https://github.com/FreeRDP/FreeRDP/wiki/Compilation
and build and installed SDL2, SDL_ttf and FreeRDP.

Versions are:
FreeRDP: 3.0.0-beta4
SDL2: 2.28.4
SDL2_ttf: 2.20.2

Little sidenode..
There is a list of "suggested base dependencies" for debian based.
There are 2 packages/libraries that are incorrect.

libkrb5-multidev should be libkrb5-dev
and
libsdl2-tty-dev should be libsdl2-ttf-dev

Am i correct here? Because i can´t find libkrb5-multidev nor libsdl-tty-def when i try sudo apt install..

Now i am a bit confused, because i can choose between xfreerdp, wlfreerdp and sdl-freerdp.
I guess sdl-freerdp is the one i should use?

Cheers
